classes/controllers/grid/feature/InfiniteScrollingFeature.inc.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+/**
+ * @file classes/controllers/grid/feature/InfiniteScrollingFeature.inc.php
+ *
+ * Copyright (c) 2014 Simon Fraser University Library
+ * Copyright (c) 2000-2014 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class InfiniteScrollingFeature
+ * @ingroup controllers_grid_feature
+ *
+ * @brief Add infinite scrolling functionality to grids. It doesn't support
+ * category grids.
+ *
+ */
+
+import('lib.pkp.classes.controllers.grid.feature.GeneralPagingFeature');
+
+class InfiniteScrollingFeature extends GeneralPagingFeature{
+
+	/**
+	 * Constructor.
+	 */
+	function InfiniteScrollingFeature() {
+		parent::GeneralPagingFeature('infiniteScrolling');
+	}
+
+
+	//
+	// Extended methods from GridFeature.
+	//
+	/**
+	 * @see GridFeature::getJSClass()
```

Comment:
Use @copydoc rather than @see unless you're re-specifying the @param / @return stuff